### ISSUE TITLE
BUG - Pokemons stop getting transfered after some hours

### ISSUE THREAD

**Issue by Deuxgod** [finndev/PokeBuddy#460]
If you leave the bot on during the night after few hours the repeated pokemons stop getting transfered leading to a inventary full. Hitting the Transfer bad pokemons on and off won't change the behaviour, just a full restard will do it.

**Comment by pokemonloverx333** [comment]
yes , not changeable , just restart and start the bot and do this : Farm Pokestops <- out Evolve Pokemons <- out  then let it run by pressing start , it should be doing nothing at first , and after first message " updating" it should transfer the double ones , after that press evolve to evolve the last high cp ones and then press farm pokestops back on , worked for me

**Comment by Deuxgod** [comment]
That would be a solution if you're available at the PC. What would be ideal would be that the bug never happened despite the time the bot is on. Thank you for your suggestion though.
